stdlib/src/collections/string.mojo

```diff
@@ -784,68 +784,25 @@ struct String(
 
     @always_inline
     @implicit
-    fn __init__(out self, owned impl: List[UInt8, *_]):
-        """Construct a string from a buffer of bytes without copying the
-        allocated data.
-
-        The buffer must be terminated with a null byte:
-
-        ```mojo
-        var buf = List[UInt8]()
-        buf.append(ord('H'))
-        buf.append(ord('i'))
-        buf.append(0)
-        var hi = String(buf)
-        ```
+    fn __init__(out self, impl: List[Byte, *_]):
```

Comment:
Why is it no longer owned? This would force making a new allocation.

Reply:
AFAIK not if you do `String(some_list^)`

Reply:
I can also just add it now :man_shrugging: